app/services/exports/funding_trainee_summary_data.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Exports
+  class FundingTraineeSummaryData
+    VULNERABLE_CHARACTERS = %w[= + - @].freeze
+
+    def initialize(trainee_summary)
+      @data = format_rows(trainee_summary)
+    end
+
+    def csv
+      header_row ||= data.first&.keys
+
+      CSV.generate(headers: true) do |rows|
+        rows << header_row
+
+        data.map(&:values).each do |value|
+          rows << value.map { |v| sanitise(v) }
+        end
+      end
+    end
+
+    def filename
+      "#{trainee_summary.payable_id}-#{trainee_summary.academic_year}-to-#{trainee_summary.academic_year(+1)}.csv"
+    end
+
+  private
+
+    attr_reader :data
+
+    def format_rows(trainee_summary)
+      trainee_summary.rows.map do |row|
+        trainee_summary_row_amount = row.amounts.first
+        {
+          "Funding type" => trainee_summary_row_amount.payment_type,
+          "Route" => row.route,
+          "Course" => row.subject,
+          "Lead school" => row.lead_school_name,
+          "Tier" => trainee_summary_row_amount.tier.presence || "Not applicable",
+          "Number of trainees" => trainee_summary_row_amount.number_of_trainees,
+          "Amount per trainee" => to_pounds(trainee_summary_row_amount.amount_in_pence),
+          "Total" => to_pounds(trainee_summary_row_amount.number_of_trainees * trainee_summary_row_amount.amount_in_pence),
+        }
+      end
+    end
+
+    def sanitise(value)
+      return value unless value.is_a?(String)
+
+      value.start_with?(*VULNERABLE_CHARACTERS) ? value.prepend("'") : value
+    end
+
+    def to_pounds(value_in_pence)
+      "£#{value_in_pence.to_f / 100}"
```

Comment:
I think it's best to use `ActionController::Base.helpers.number_to_currency` here because it handles certain edge cases, like not dropping the `0` when the `value_in_pence` is a multiple of ten and putting commas in the output, e.g.

```
irb(main):010:0> amount_in_pence = 123412300
=> 123412300
irb(main):011:0> ActionController::Base.helpers.number_to_currency(amount_in_pence.to_d / 100, unit: "£")
=> "£1,234,123.00"
irb(main):012:0> "£#{amount_in_pence.to_f / 100}"
=> "£1234123.0"
irb(main):013:0> 
```